python/cuml/fil/fil.pyx

```diff
@@ -245,10 +246,56 @@ cdef class ForestInference_impl():
 
 class ForestInference(Base):
     """
+    ForestInference provides GPU-accelerated inference (prediction)
+    for random forest and boosted decision tree models.
+
+    This module does not support training models. Rather, users should
+    train a model in another package and save it in a
+    treelite-compatible format. (See https://github.com/dmlc/treelite)
+    Currently, LightGBM and XGBoost GBDT and random forest models are
```

Comment:
It's a very small thing but i think it might be nice to have what GBDT stands for, just in case a user does not know.
Also I might be misunderstanding it but i thought we will be adding the support for RF in the next release?

Reply:
RF support is present in FIL (for xgboost RF), though we haven't added it to tests yet... should do this ASAP: https://github.com/rapidsai/cuml/pull/922